redbot/cogs/trivia/data/lists/minecraft.yml

```diff
@@ -0,0 +1,114 @@
+ - "torch"
+"What mob dances when music played?":
+ - "parrot"
+ - "parrots"
+"What is the name of the most popular arcade and minigame server?":
+ - "hypixel"
+"What is the default port of a Minecraft server?":
+ - "25565"
+"This block was added in 1.19, and emits small amounts of light:":
+ - "glow lichen"
+"Which item besides cactus, when planted, pricks you when stepped on?":
+ - "sweet berries"
+"What item is needed to make green dye?":
+ - "cactus"
+"What ore is only found in the nether, and grants decent amounts of XP?":
+ - "quartz"
+"What ore drop is needed to enchant items in an enchantment table? (full name)":
+ - "lapis lazuli"
+"An enderman gets angry if you look at its:":
+ - "eyes"
+"What mob infects stone?":
+ - "silverfish"
+"This mob makes you fly when you are hit by its projectile:":
+ - "shulker"
+"This container allows you to access storage across any dimension, and its contents are unique to the player who opened it:":
+ - "enderchest"
+ - "ender chest"
+"Which item, when placed in anvil allows you to repair damaged/broken elytras?":
+ - "phantom membrane"
+"This ORE and its successors do not burn in lava:":
+ - "ancient debris"
+"To place a chest next to another without joining them, you must do what type of click?":
+ - "shift click"
+ - "shiftclick"
+ - "shift rightclick"
+ - "shift right click"
+"To stop an enderman from attacking you, you must wear a:":
+ - "jackolantern"
+ - "jack 'o lantern"
+ - "jack-o-lantern"
+ - "carved pumpkin"
+"Name an ingredient used in the crafting of regular fireworks:":
+ - "paper"
+ - "gunpowder"
+"To prepare a zombie villager to be overturned, you must splash it with this type of potion:":
+ - "weakness"
+"The creeper was made after a failed attempt to create which animal?":
+ - "pig"
+"Which flying hostile mob burns during the daytime?":
+ - "phantom"
+"Which enchant does more damage to spiders, endermite, bees, silverfish, and endermites?":
+ - "bane of arthropods"
+"This enchant for projectile weapons lights the entities it hits on fire:":
+ - "flame"
+"This mob holds and collects more of the item you give it:":
+ - "allay"
+"The warden spawns in which biome:":
+ - "ancient city"
+"To get a villager that trades carrots, wheat, and other farm foods, you must place which profession block near it?":
+ - "composter"
+"This item is used as currency for villagers:":
+ - "emerald"
+"This block, when hit with a projectile, emits redstone power:":
+ - "target"
+"Which block extends and retracts, while keeping hold of the block attached to it?":
+ - "sticky piston"
+"Wool is commonly used in redstone builds. This is because redstone signals can...":
+ - "pass through it"
+"Besides bedrock, this block cannot be broken.":
+ - "end portal frame"
+"A renewable source of fuel in Minecraft is:":
+ - "charcoal"
+"This block spreads like an infection throughout your world:":
+ - "skulk catalyst"
+"What is the fastest type of tool in the game?":
+ - "golden"
+ - "gold"
```

Comment:
This is not completely accurate. There are a few blocks where golden tools are slower than than others, such as when you mine blocks like Obsidian,  Blocks of Any Ore, and Gold Ore.